src/main/java/io/excaliburfrc/robot/subsystems/LEDS.java

```diff
@@ -0,0 +1,34 @@
+package io.excaliburfrc.robot.subsystems;
+
+import edu.wpi.first.wpilibj.PWM;
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.RunCommand;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import io.excaliburfrc.robot.Constants.LedsConstants;
+
+public class LEDS extends SubsystemBase {
+  private final PWM leds = new PWM(LedsConstants.LEDS_PORT);
+
+  public enum LedMode {
+    BLUE(0.87),
+    RED(0.61),
+    GREEN(0.73),
+    YELLOW(0.67),
+    RAINBOW(-0.97),
+    OFF(0.99);
+
+    LedMode(double c) {
+      color = c;
+    }
+
+    public final double color;
+  }
+
+  public Command setColor(LedMode color) {
+    return new RunCommand(() -> leds.setSpeed(color.color), this);
```

Comment:
I'm not sure that a `RunCommand` is the best way to do this. I can't outright think of a better way though.

Reply:
maybe it should just return void and we'll call the set color function in the `Periodic` function?

Reply:
like this:

`  private double LEDColor;
  
  public void setColor(LedMode color) {
    LEDColor = color.dutyCycle;
  }

  @Override
  public void periodic() {
    leds.setSpeed(LEDColor);
  }`

Reply:
That's not what I'm not sure about. I'm thinking about how the requirements should work.